jobs/sensu.rb

```diff
@@ -100,80 +96,42 @@ def get_assoc(column_name, entry)
     events.push(*current_env)
   end
 
-  warn = []
-  crit = []
   # status = get_data(SENSU_API_ENDPOINT, '/status',
 
-  columns['config']['default'].each do |column|
-    hrows[0][:cols].insert(-1, value: column)
+  all_columns = ['hostname', 'status', 'name', 'handlers', 'output', 'history', 'team',
+                 'occurances', 'subscribers', 'command']
+
+  all_columns.each do |column|
+    hrows.push(column)
   end
 
   # for each event...
+  all_data = []
   events.each_with_index do |event, _event_index|
     status = event['check']['status']
     status_string = get_status(status)
     event_var = { cols: [] }
 
     # for each column....
-    columns['config']['default'].each_with_index do |column, _column_index|
+    data_list=[]
+    all_columns.each_with_index do |column, _column_index|
       data = get_assoc(column, event)
       data = data.nil? ? '' : data.chomp # remove tailing whitespace
 
       # add column to event var
       column_var = { class: status_string, value: data }
       event_var[:cols].insert(-1, column_var)
-    end
```

Comment:
logic has been moved to the widget, here we are now just sending the raw data in a simple structure so that it can be easily parsed later